test/integrations/sources/adjust/data.ts

```diff
@@ -0,0 +1,122 @@
+import utils from '../../../../src/v0/util';
+
+const defaultMockFns = () => {
+  jest.spyOn(utils, 'generateUUID').mockReturnValue('97fcd7b2-cc24-47d7-b776-057b7b199513');
+};
+
+export const data = [
+  {
+    name: 'adjust',
```

Comment:
All test cases should have id property

Reply:
will add this in next release as it is a minor issue.